dune_client/base_client.py

```diff
@@ -0,0 +1,43 @@
+""""
+Basic Dune Client Class responsible for refreshing Dune Queries
+Framework built on Dune's API Documentation
+https://duneanalytics.notion.site/API-Documentation-1b93d16e0fa941398e15047f643e003a
+"""
+from __future__ import annotations
+
+import logging.config
+from json import JSONDecodeError
+from typing import Any, Union
+
+from aiohttp import ClientResponse
+from requests import Response
+
+
+# pylint: disable=too-few-public-methods
+class BaseDuneClient:
```

Comment:
added base client which basically has logger, some constants and common function for unwrapping response

Reply:
I like this, however, do you think we could make the log level configurable (and not hard coded as DEBUG).

Reply:
we could, also we can leave it as default and put logger.info/logger.debug where its suitable

Reply:
Ok, so I just realized that you actually removed the hard-coded DEBUG log level:

```
log = logging.getLogger(__name__)
logging.basicConfig(
    format="%(asctime)s %(levelname)s %(name)s %(message)s", level=logging.DEBUG
)
```
So, this is fine as is! Sorry for the confusion here.